TestPlaybooks/playbook-XFE_Playbook_Test.yml

```diff
@@ -740,14 +806,326 @@ tasks:
+      long: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 3880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "30":
+    id: "30"
+    taskid: 39f82492-e240-4af3-85e6-6ec6a357a3dc
+    type: condition
+    task:
+      id: 39f82492-e240-4af3-85e6-6ec6a357a3dc
+      version: -1
+      name: verifyCSV
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      "yes":
+      - "32"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: http://www.iuqerfsodp9ifjaposdfjhgosurijfaewrwergwea.com/
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: https://exchange.xforce.ibmcloud.com/
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: 103.225.52.112
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: 8.8.8.8
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: https://www.demisto.com
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: https://www.google.com/
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: 4b41a3475132bd861b30a878e30aa56a
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: DBotScore.Indicator
+            iscontext: true
+          right:
+            value:
+              simple: 37c919f28798a886ec7202cc39818b8ce641ae83
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 4220
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "31":
+    id: "31"
+    taskid: 77126f76-4940-4fab-8bcd-868fbe30b4f0
+    type: regular
+    task:
+      id: 77126f76-4940-4fab-8bcd-868fbe30b4f0
+      version: -1
+      name: check domain with CSV
+      description: Check domain reputation
+      script: XFE|||domain
```

Comment:
please verify both domains are in `Domain.Name`

Reply:
It output Domain.Name only when there is a Malicious
(The same is also in the ip - file - url command)

Reply:
ok, thats not how it should be
lets change it to always output `Domain.Name`, and do same for the indicators